counterparty-core/counterpartycore/test/fixtures/api_v2_fixtures.json

```diff
@@ -1,5 +1,5 @@
 {
-    "http://api:api@localhost:10009/blocks": {
+    "http://rpc:rpc@localhost:10009/v2/blocks": {
```

Comment:
We could remove the basic auth here, no?

Reply:
yep. done.